src/main/resources/application.properties

```diff
@@ -1,10 +1,6 @@
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
-spring.datasource.url=jdbc:mysql://localhost:3306/tasks
-spring.datasource.username=serg
-spring.datasource.password=22062020
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
return these variables for local running. you can use variables from env instead of values